src/histryExtension.ts

```diff
@@ -0,0 +1,114 @@
+import * as vscode from "vscode"
+
+import { getNonce } from "./getNonce"
+
+export class HistoryPanel {
```

Comment:
I have some ideas how to make this less static but still use the singleton pattern.